### ISSUE TITLE
Support Android Test Orchestrator with Spoon

### ISSUE THREAD

**Issue by jaredsburrows** [square/spoon#497]
https://developer.android.com/training/testing/junit-runner.html#using-android-test-orchestrator

**Comment by ArazAbishov** [comment]
I think @JakeWharton has already implemented something similar here: https://github.com/square/spoon/pull/456. The difference in semantics is that spoon's implementation does not reset the disk state, while orchestrator does.

**Comment by dweebo** [comment]
Yes that difference is why I'd like to use the orchestrator with spoon.  If I find time I'd like to play around with implementing this.

**Comment by jaredsburrows** [comment]
@dweebo Sounds good. Who can we contact to get Spoon 2 released?

**Comment by dweebo** [comment]
I did a few pull requests last month and @edenman merged them in.  I guess him or @JakeWharton

**Comment by JakeWharton** [comment]
If it were done, it would have been released.

**Comment by dweebo** [comment]
@JakeWharton are there specific items outstanding before 2.0 is done?  Happy to help towards that goal.

**Comment by jaredsburrows** [comment]
Yes. If there are outstanding items, please let us know so we can contribute.

**Comment by anandkiran2007** [comment]
Is this still in Open state , is there any work planned to add support for Spoon to support Android Test Orchestrator ?

**Comment by kaustubhkp** [comment]
@JakeWharton can you please confirm it's still open or fix is release ? - thank you

**Comment by connerT** [comment]
Is this project being abandoned? I will look for another solution if so. Just curious.

**Comment by JakeWharton** [comment]
It was abandoned like 6 years ago, yes.
